examples/customresources/models/mysum/mysum.go

```diff
@@ -15,6 +15,15 @@ import (
 // Model is the full model definition.
 var Model = resource.NewModel("acme", "demo", "mysum")
 
+// Config is the sum model's config.
+type Config struct {
+	Subtract bool `json:"subtract,omitempty"`
```

Comment:
json tag comments needed. Here need to explain that omitempty is gonna be "false" by default as a special case.